src/bot.ts

```diff
@@ -232,7 +236,7 @@ export class DiscordBot {
     if (!Array.isArray(msg)) {
       msg = [msg];
     }
-    msg.forEach((m: Discord.Message) => {
+    await Bluebird.each(msg, (m: Discord.Message) => {
```

Comment:
For some reason tslint is claiming this was a non-await promise....do you know what's going on? Maybe making this a normal for...of loop with `await`'ing the store insert would do, too? (it seems to fix linting)

```
    for (const m of msg) {
      log.verbose("DiscordBot", "Sent ", m);
      this.sentMessages.push(m.id);
      const evt = new DbEvent();
      evt.MatrixId = event.event_id + ";" + event.room_id;
      evt.DiscordId = m.id;
      // Webhooks don't send guild info.
      evt.GuildId = guildId;
      evt.ChannelId = channelId;
      await this.store.Insert(evt);
    }
```

Thank you for the PR, though, it looks great otherwise!